File: Data Structures & Algorithms/find-pivot-index/submission-0.py
class Solution:
    def pivotIndex(self, nums: list[int]) -> int:
        left_sum = 0
        right_sum = 0

        for i in range(len(nums)):
            right_sum = 0
            for j in nums[i+1:]:
                right_sum += j
            print(right_sum)
            if right_sum == left_sum:
                return i
            left_sum += nums[i]

        return -1

File: Data Structures & Algorithms/move-zeroes/submission-0.java
class Solution {
    public void moveZeroes(int[] nums) {
        int zeroCount = 0;
        int l = 0;
        for (int i = 0; i < nums.length; i++) {
            if (nums[i] == 0) {
                zeroCount++;
            } else {
                nums[l++] = nums[i];
            }
        }
        int k = nums.length - 1;
        while (zeroCount > 0) {
            nums[k--] = 0;
            zeroCount--;
        }
    }
}
